index.d.ts

```diff
@@ -676,6 +676,19 @@ declare module 'egg' {
 
     truncated: boolean;
   }
+                   
+  interface GetFileStreamOptions {
+    requireFile?: boolean // required file submit, default is true
+    defCharset?: string
+    limits?: any
+    checkFile?(
+      fieldname: string,
+      file: any,
+      filename: string,
+      encoding: string,
+      mimetype: string
+    ): any
```

Comment:
Return type of `checkFile` field should be `void | Error`.  see https://github.com/cojs/busboy/blob/master/index.js#L101

The type of `limits` field is

![image](https://user-images.githubusercontent.com/5856440/53465958-e433ab00-3a8a-11e9-912c-013502a9e803.png)

see https://github.com/mscdex/busboy#busboy-methods